src/ContainerIntegration.jsx

```diff
@@ -97,7 +97,7 @@ const ContainerIntegration = ({ container, containerDetail, localImages }) => {
     const volumes = renderContainerVolumes(containerDetail.Mounts);
 
     const image = localImages.filter(img => img.Id === container.ImageID)[0];
-    const env = renderContainerEnv(containerDetail.Config.Env, image.Env || []);
+    const env = <ContainerEnv containerEnv={containerDetail.Config.Env} imageEnv={image.Env || []} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)